Source/Core/Core/HW/DSPHLE/DSPHLE.cpp

```diff
@@ -191,26 +191,37 @@ void DSPHLE::DSP_WriteMailBoxLow(bool cpu_mailbox, u16 value)
 // Other DSP functions
 u16 DSPHLE::DSP_WriteControlRegister(u16 value)
 {
-  DSP::UDSPControl temp(value);
```

Comment:
Yeah. As a general practice, when doing multiple operations to a single member variable, the compiler can often do a better job if you use local variables instead.